### PR TITLE
[stable/rabbitmq-ha] Use a more specific docker image tag

### PR DESCRIPTION
**Pull request by max-rocket-internet**
#### What this PR does / why we need it:

Setting a specific tag instead of the general `3.7` one. The reason is that otherwise you can end up with different versions running in the same cluster when the pods are started at different times. Granted the time between pod starts need to be quite large but still.

For example we had 3.7.10, 3.7.11 and 3.7.12 all running together ⚠

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.8
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Bump the appVersion as well

File: stable/rabbitmq-ha/README.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ and their default values.
...
 | `image.repository`                             | RabbitMQ container image repository                                                                                                                                                                   | `rabbitmq`                                                 |
-| `image.tag`                                    | RabbitMQ container image tag                                                                                                                                                                          | `3.7-alpine`                                               |
+| `image.tag`                                    | RabbitMQ container image tag                                                                                                                                                                          | `3.7.11-alpine`                                            |
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/README.md]: Can you switch to the latest 3.7.12-alpine?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/README.md (hunk 2 above)
Comment (max-rocket-internet) [reply stable/rabbitmq-ha/README.md]: OK, sure

**Comment by max-rocket-internet** [comment]
@steven-sheehy changes done

**Comment by steven-sheehy** [comment]
/assign
Sorry, can you rebase to fix conflict?

**Comment by max-rocket-internet** [comment]
Done

**Comment by steven-sheehy** [comment]
/lgtm
Thanks!

**Pull request merged**